src/Components/Home/Home.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+import "./Home.css";
+import ApiDetails from "../ApiDetails/ApiDetails";
+import { getAllCategories, getCategoryEntries } from "../../apiCalls";
+
+
+class Home extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            categories: [],
+            focus: {}
+        }
+
+    }
+
+    componentDidMount() {
+
+        this.setState(getAllCategories())
+        console.log(this.state.categories)
```

Comment:
@Estaffieri is this console logging as intended?

Reply:
This was experimental and should have been removed prior to pushing.

Reply:
I have removed this console.log.